Makefile

```diff
@@ -0,0 +1,20 @@
+lint:
```

Comment:
Suggest adding `.PHONY: lint protos ...`

Also note that since `lint` is your first target, `make` within the root will default to `lint`.

Reply:
Done. Switched so `gen` is the first item.